### PR TITLE
Fix help

### LINKED ISSUE #594: "What is this?" for Notes is blank

### PR DESCRIPTION
**Pull request by erose**
Fixes #594, #587, #583.

### CODE REVIEW

**Comment by erose** [comment]
I don't know CSS, so would love someone to look at my CSS!

**Comment by Jrafalak** [comment]
I'm not an expert in CSS, but it looks good to me. [Here](https://css-tricks.com/currentcolor/) is the link we were looking at about currentColor at CFB.

File: app/assets/javascripts/student_profile/help_bubble.js [hunk 1]
```diff
@@ -7,11 +7,13 @@
   var HelpBubble = window.shared.HelpBubble = React.createClass({
     propTypes: {
       title: React.PropTypes.string.isRequired, // e.g. 'What is a Note?'
-      content: React.PropTypes.string.isRequired, // HTML string which will be rendered in the modal's box.
+      content: React.PropTypes.object.isRequired, // React DOM objects which will be displayed in the modal text box.
```
Comment (kevinrobinson) [inline app/assets/javascripts/student_profile/help_bubble.js]: FYI, you can use `React.PropTypes.element` to indicate it has to be a React element like your good comment says :)

**Comment by kevinrobinson** [comment]
👍 this css looks like it's doing the trick, so I think you know more than you realize :)

**Pull request merged**